pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
I'm sorry, I don't quite understand what you mean. Could you please elaborate on the trade-offs?